jax/experimental/jax2tf/README.md

```diff
@@ -3,57 +3,72 @@
-In practice this means that you can take some code written in JAX and execute it using
-TensorFlow eager mode, or stage it out as a TensorFlow graph, even use it
-with TensorFlow tooling such as: SavedModel for archival ([examples below](#usage-saved-model)),
-TensorFlow Serving ([examples](https://github.com/google/jax/blob/main/jax/experimental/jax2tf/examples/serving/README.md)),
-TFX ([examples](https://github.com/tensorflow/tfx/blob/master/tfx/examples/penguin/README.md#instructions-for-using-flax)),
-TensorFlow Lite ([examples](https://github.com/google/jax/blob/main/jax/experimental/jax2tf/examples/tflite/mnist/README.md)),
-TensorFlow.js ([examples](https://github.com/google/jax/blob/main/jax/experimental/jax2tf/examples/tf_js/quickdraw/README.md)),
-or TensorFlow Hub.
-
-This package also contains the `jax2tf.call_tf` mechanism to call TensorFlow functions
-from JAX. These functions can be called in JAX's op-by-op execution mode,
-in which case the callee is executed in TensorFlow eager mode, or in JAX's jit (staged) context,
-in which case the callee is compiled to XLA and embedded in JAX's lowered HLO.
-
-Both interoperation directions rely on the ability of
-TensorFlow to use the XLA compiler (`tf.function(jit_compile=True)`). For the
-`jax2tf.convert` direction the JIT compilation of the resulting TensorFlow code ensures
-that the performance characteristics of the code match those of the JAX source.
-For the `call_tf` direction, JIT compilation is an essential part of the implementation
-mechanism. Only TensorFlow functions that can be JIT-compiled can be called from
-JAX in a jit context.
-Since the TensorFlow functions that are produced by `jax2tf.convert` can
-be JIT-compiled by design, we can call them using `jax2tf.call_tf` thus achieving
-a round-trip from JAX to TensorFlow (e.g., a SavedModel) and back.
+These APIs can be called in sequence, e.g., to reload in JAX a program that
+has been serialized from JAX to a TensorFlow SavedModel, or to save to
+TensorFlow SavedModel a JAX program that uses a TensorFlow library.
+
+These APIs can be used in different modes, but in the default preferred mode
+the interoperation is by way of a StableHLO/HLO intermediate form. That is,
+the target function is lowered to HLO using standard native JAX or TensorFlow APIs,
+and then the HLO is invoked from TensorFlow or JAX. This has the advantage that
+it stays faithful to the semantics of the native target function, even as JAX and
+TensorFlow evolves. We refer to this usage mode as "native lowering" or
+"native serialization".
+
+Note that at the moment when using JAX native serialization the whole
+JAX compilation unit is wrapped with a single thin TensorFlow op, called `XlaCallModule`,
+that carries a serialized version of the HLO obtained from JAX. This
+op simply compiles the HLO and invokes it.
+While this may feel as not purely native, it has several advantages:
```

Comment:
Instead of saying "this may feel as not purely native", we can just say "The reason we still wrap the lowered HLO in a TensorFlow op is..."